### ISSUE TITLE
Should not lint on whole module imported R6 class

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#29]
## box.linters version

0.0.0.9006

## Source code to lint

```R
box::use(
  path/to/module_r6,
)

s <- module_r6$SomeClass$new()
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Lint result

```
<text>:3:3: warning: [box_unused_attached_mod_linter] Attached module unused.
  path/to/module_r6,
  ^~~~~~~~~~~~~~~~~
<text>:6:6: warning: [box_usage_linter] <package/module>$function does not exist.
s <- module_r6$SomeClass$new()
     ^~~~~~~~~~~~~~~~~~~~~~~
```

## Expected result

- [ ] Should not lint on "unused module"
- [ ] Should not lint on `module$r6_class$new()`
